[INFO   ] Logger: Record log in /storage/emulated/0/github/FDS/.kivy/logs/kivy_18-12-26_90.txt
[INFO   ] Kivy: v1.10.1.dev0, git-Unknown, 20171218
[INFO   ] Python: v2.7.14 (default, Dec 18 2017, 09:30:59) 
[GCC 7.2.0]
[INFO   ] Factory: 194 symbols loaded
[INFO   ] Image: Providers: img_tex, img_dds, img_sdl2, img_gif (img_pil, img_ffpyplayer ignored)
[INFO   ] Text: Provider: sdl2
[INFO   ] KivyMD: KivyMD version: 0.8.6
[INFO   ] OSC: using <thread> for socket
[INFO   ] Window: Provider: sdl2
[INFO   ] GL: Using the "OpenGL ES 2" graphics system
[INFO   ] GL: Backend used <gl>
[INFO   ] GL: OpenGL version <OpenGL ES 3.2 V@269.0 (GIT@908a5ce, I77d3059488) (Date:06/07/18)>
[INFO   ] GL: OpenGL vendor <Qualcomm>
[INFO   ] GL: OpenGL renderer <Adreno (TM) 506>
[INFO   ] GL: OpenGL parsed version: 3, 2
[INFO   ] GL: Texture max size <16384>
[INFO   ] GL: Texture max units <16>
[INFO   ] Window: auto add sdl2 input provider
[INFO   ] Window: virtual keyboard not allowed, single mode, not docked
[INFO   ] GL: NPOT texture support is available
[WARNING] stderr: Traceback (most recent call last):
[WARNING] stderr:   File "/storage/emulated/0/github/FDS/main.py", line 495, in <module>
[WARNING] stderr:     
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid2/files/arm-linux-androideabi/lib/python2.7/site-packages/kivy/app.py", line 803, in run
[WARNING] stderr:     root = self.build()
[WARNING] stderr:   File "/storage/emulated/0/github/FDS/main.py", line 473, in build
[WARNING] stderr:     self.main_widget.ids.box.remove_widget(botao)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid2/files/arm-linux-androideabi/lib/python2.7/site-packages/kivy/lang/builder.py", line 382, in load_string
[WARNING] stderr:     self._apply_rule(widget, parser.root, parser.root)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid2/files/arm-linux-androideabi/lib/python2.7/site-packages/kivy/lang/builder.py", line 567, in _apply_rule
[WARNING] stderr:     self._apply_rule(child, crule, rootrule)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid2/files/arm-linux-androideabi/lib/python2.7/site-packages/kivy/lang/builder.py", line 567, in _apply_rule
[WARNING] stderr:     self._apply_rule(child, crule, rootrule)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid2/files/arm-linux-androideabi/lib/python2.7/site-packages/kivy/lang/builder.py", line 566, in _apply_rule
[WARNING] stderr:     self.apply(child)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid2/files/arm-linux-androideabi/lib/python2.7/site-packages/kivy/lang/builder.py", line 451, in apply
[WARNING] stderr:     self._apply_rule(widget, rule, rule, ignored_consts=ignored_consts)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid2/files/arm-linux-androideabi/lib/python2.7/site-packages/kivy/lang/builder.py", line 567, in _apply_rule
[WARNING] stderr:     self._apply_rule(child, crule, rootrule)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid2/files/arm-linux-androideabi/lib/python2.7/site-packages/kivy/lang/builder.py", line 567, in _apply_rule
[WARNING] stderr:     self._apply_rule(child, crule, rootrule)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid2/files/arm-linux-androideabi/lib/python2.7/site-packages/kivy/lang/builder.py", line 567, in _apply_rule
[WARNING] stderr:     self._apply_rule(child, crule, rootrule)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid2/files/arm-linux-androideabi/lib/python2.7/site-packages/kivy/lang/builder.py", line 567, in _apply_rule
[WARNING] stderr:     self._apply_rule(child, crule, rootrule)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid2/files/arm-linux-androideabi/lib/python2.7/site-packages/kivy/lang/builder.py", line 567, in _apply_rule
[WARNING] stderr:     self._apply_rule(child, crule, rootrule)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid2/files/arm-linux-androideabi/lib/python2.7/site-packages/kivy/lang/builder.py", line 567, in _apply_rule
[WARNING] stderr:     self._apply_rule(child, crule, rootrule)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid2/files/arm-linux-androideabi/lib/python2.7/site-packages/kivy/lang/builder.py", line 526, in _apply_rule
[WARNING] stderr:     cls = Factory_get(cname)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid2/files/arm-linux-androideabi/lib/python2.7/site-packages/kivy/factory.py", line 131, in __getattr__
[WARNING] stderr:     raise FactoryException('Unknown class <%s>' % name)
[WARNING] stderr: kivy.factory.FactoryException: Unknown class <AvatarSampleWidget>
